package.json

```
@@ -0,0 +1,91 @@
+{
+  "name": "manh-react-survey",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "axios": "^1.4.0",
```

please use the exact version for all the dependencies in this file

```suggestion
    "axios": "1.4.0",
```